configs/opencv-ovms/scripts/run_gst.sh

```diff
@@ -36,7 +36,7 @@ else
 fi
 
 echo $rmDocker
-TAG=sco-soc:2.0
+TAG=dlstreamer:2.0
```

Comment:
sorry i missed this in my PR: this line can be removed, this env is no longer used.